setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = pytd
-version = 1.7.0
+version = 1.7.1
```

Comment:
IIRC, the version number should be changed in another PR before release.

@tung-vu-td can you tell us the right way?

Reply:
Yeah, I wasn't sure whether this repo bunches up a few features/bugfixes per minor patch or bumps it with every PR. Whatever is the preferred option here

Reply:
You can see in the previous release PR https://github.com/treasure-data/pytd/pull/140/files

It looks like the version number is bumped on the release PR. Let's revert this line.

Reply:
Reverted line change - thanks for checking!

Reply:
@chezou Yes, usually (for other client tools as well), the version is bumped on release PR as you mentioned, making it easier to track